contracts/Marketplace.sol

```diff
@@ -46,6 +46,11 @@ contract Marketplace is MetaTxContext, FreeportDelegator, HasGlobalNftId {
         uint256 price,
         uint256 amount);
 
+    /**
+     * @dev Emitted when `value` tokens of token type `id` are transferred from `from` to `to` by `operator`.
+     */
+    event TransferSingle(address indexed operator, address indexed from, address indexed to, uint256 id, uint256 value);
```

Comment:
That event belongs on the Collection, not Marketplace.

Reply:
Yes, it is, but we can not listen on all collections and still need this events